src/test/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiServiceSpec.scala

```diff
@@ -21,7 +21,7 @@ class AdminApiServiceSpec extends ApiServiceSpec {
 
   case class TestApiService(dataSource: SlickDataSource, gcsDAO: MockGoogleServicesDAO)(implicit val executionContext: ExecutionContext) extends ApiServices with MockUserInfoDirectives
 
-  def withApiServices(dataSource: SlickDataSource)(testCode: TestApiService => Any): Unit = {
+  def withApiServices[T](dataSource: SlickDataSource)(testCode: TestApiService =>  T): T = {
```

Comment:
I would really like to refactor the duplicated code here, but it's nontrivial so I'm leaving that for another day: https://broadinstitute.atlassian.net/browse/GAWB-790